assets/js/components/settings/EntityOwnershipChangeNotice.js

```diff
@@ -44,13 +44,15 @@ export default function EntityOwnershipChangeNotice( { slug } ) {
 	const slugs = Array.isArray( slug ) ? slug : [ slug ];
 
 	const storeNames = useSelect( ( select ) => {
-		const { getModuleStoreName } = select( CORE_MODULES );
+		const { getModuleStoreName, getSharedRoles } = select( CORE_MODULES );
 
 		return slugs.reduce( ( acc, currentSlug ) => {
 			const storeName = getModuleStoreName( currentSlug );
-			if ( storeName ) {
-				acc[ currentSlug ] = storeName;
+
+			if ( storeName && !! getSharedRoles( currentSlug )?.length ) {
```

Comment:
While this works, it's a bit unnecessarily tied into the derivation of `storeNames`. Can we make it a bit more explicit that we're filtering out modules that don't have any shared roles from being included here?

Reply:
Ah, fair enough. I'll refactor it a bit 👍🏻